app/modules/details/DetailsScreenStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { StyleSheet } from 'react-native';
+import {
+  Colors,
+  horizontalScale,
+  moderateScale,
+  verticalScale,
+} from '../../theme';
+import { Strings } from '../../constants';
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+    backgroundColor: Colors.white,
+  },
+  top: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  overlay: {
+    width: '100%',
+    height: '100%',
+    backgroundColor: Colors.black + Strings.opacity,
```

Comment:
we can define color with opacity inside the color file itself (e.g. const blackWithOpacity = '#color-hexcode90' )
- last two digits will be based on the value of the opacity.